app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -210,6 +219,8 @@ class PatientSummaryScreen :
 
   private val additionalEvents = DeferredEventSource<PatientSummaryEvent>()
 
+  private val shouldShowStatinNudge = mutableStateOf(false)
```

Comment:
FYI, You can use `by` instead of `=` here and we can change to `var`, allows us to directly use variable as state rather than having to `state.value`